ui/raidboss/raidboss_config.js

```diff
@@ -907,7 +907,7 @@ UserConfig.registerOptions('raidboss', {
           '朝鲜语 (ko)': 'ko',
         },
         ko: {
-          'FFXIV Plugin 언어 사용': 'default',
+          '주 사용 언어 사용': 'default',
```

Comment:
Can you remove the de/fr/cn sections above just so folks know that they need to be translated? I'd rather them be missing than be incorrect, because nobody will notice if they are just incorrect.

Alternatively, maybe @Akurosia @VanSlanzar @maiko-tan-coding @ShadyWhite can chime in with translations before this gets merged? (Sorry for ping!)

Reply:
need correction line 883 
` 'Utiliser la langue d\'affichage : 'par défaut',`
For the rest, it's good.

Reply:
Other folks can follow up with pull requests for translations in other languages (sorry).